yarn-project/archiver/src/archiver/memory_archiver_store/memory_archiver_store.ts

```diff
@@ -427,24 +348,20 @@ export class MemoryArchiverStore implements ArchiverDataStore {
    * Gets the number of the latest L2 block processed.
    * @returns The number of the latest L2 block processed.
    */
-  public getBlockNumber(): Promise<number> {
+  public getSynchedL2BlockNumber(): Promise<number> {
     if (this.l2BlockContexts.length === 0) {
       return Promise.resolve(INITIAL_L2_BLOCK_NUM - 1);
     }
     return Promise.resolve(this.l2BlockContexts[this.l2BlockContexts.length - 1].block.number);
   }
 
-  public getL1BlockNumber() {
-    const addedBlock = this.l2BlockContexts[this.l2BlockContexts.length - 1]?.block?.getL1BlockNumber() ?? 0n;
-    const newMessages = this.lastL1BlockNewMessages;
-    const addedMessages = this.lastL1BlockAddedMessages;
-    const cancelledMessages = this.lastL1BlockCancelledMessages;
+  public getSynchedL1BlockNumbers(): Promise<ArchiverL1SynchPoint> {
```

Comment:
The `s` on the function name is quite confusing for me, at it makes it sound like a list

Reply:
Took a note and will address in a separate PR.